stories/Interactive.stories.js

```diff
@@ -0,0 +1,16 @@
+import { storiesOf } from '@storybook/vue'
+import { action } from '@storybook/addon-actions'
+import { linkTo } from '@storybook/addon-links'
+
+import ModalInteractive from '../ozaria/site/components/interactive/PageInteractive/common/ModalInteractive.vue'
+import BaseButton from '../ozaria/site/components/interactive/PageInteractive/common/BaseButton.vue'
+
+storiesOf('ModalInteractive', module).add('pure vue modal', () => ({
+  components: { ModalInteractive },
+  template: '<modal-interactive />'
+}))
+
+storiesOf('BaseButton', module).add('pure vue modal', () => ({
```

Comment:
Text typo `'pure vue modal'` should be `pure vue button`?

Reply:
Fixed